workflow/usage.rst

```diff
@@ -229,11 +229,10 @@ Available events
 
 The following events are dispatched for all workflows:
 
-* ``workflow.guard``: occurs just before starting a transition. It allows you to
-  prevent the transition by calling ``$event->setBlocked(true);`` as shown above.
-* ``workflow.leave``: occurs just after an object has left it's current state.
-* ``workflow.transition``: occurs just before starting to transition to the new state.
-* ``workflow.enter``: occurs just after the object has entered into the new state.
+* ``workflow.guard``: occurs just before a transition is started and when testing which transitions are available. It allows you to define that the transition is not allowed by calling ``$event->setBlocked(true);`` as shown above.
+* ``workflow.leave``: carries the marking with the initial places, occurs just after an object has left it's current state.
```

Comment:
`it's` => `its`.

To avoid confusion, maybe we should not use "initial" and say something like:
`carries the marking with the places held before an object leaves it's current state.`